README.md

```diff
@@ -44,9 +44,13 @@ After starting the bootstrap have a long coffee...
 
 ### Adding EESSI overlay
```

Comment:
While looking at your changes, I also noticed that the title of this section does not really fit anymore: it does more than just adding the overlay. So maybe this fits better?

```suggestion
### Adding the EESSI overlay and packages
```